climada/engine/impact_calc.py

```diff
@@ -238,18 +252,22 @@ def _return_empty(self, save_mat):
         return Impact.from_eih(self.exposures, self.impfset, self.hazard,
                         at_event, eai_exp, aai_agg, imp_mat)
 
-    def minimal_exp_gdf(self, impf_col):
+    def minimal_exp_gdf(self, impf_col, reassign_centroids):
         """Get minimal exposures geodataframe for impact computation
 
         Parameters
         ----------
         exposures : climada.entity.Exposures
         hazard : climada.Hazard
-        impf_col: str
-            name of the impact function column in exposures.gdf
-
-        """
-        self.exposures.assign_centroids(self.hazard, overwrite=False)
+        impf_col : str
+            Name of the impact function column in exposures.gdf
+        reassign_centroids : bool
+            Indicates whether centroids are re-assigned to the self.exposures object
+            or kept from previous impact calculation with a hazard of the same hazard type.
+            Centroids assignment is an expensive operation, set this to true if you know that
+            the centroids are the same between two impact calculations.
```

Comment:
Same as above
```suggestion
            Centroids assignment is an expensive operation; set this to ``False`` to save 
            computation time if the centroids have not changed since the last impact
            calculation.
```